fixieai/agents/code_shot_test.py

```diff
@@ -24,10 +24,18 @@
 """
 
 
+@pytest.fixture(autouse=True)
+def mock_token_verifier():
```

Comment:
nit: The pytest friendly way of mocking is to use the mocker fixture:
```python
def mock_token_verifier(mocker):
    return mocker.patch(
        "fixieai.agents.code_shot._VerifiedTokenClaims.from_token",
        return_value=code_shot._VerifiedTokenClaims(agent_id="fake_agent_id")
    )
```

Reply:
Ah, I didn't realize we had `pytest-mock`, thank you!

In concert with removing the api URL from `FixieClient` I also normalized the `requests_mock` usage.